composer.json

```diff
@@ -116,7 +115,8 @@
         "webfontkit/open-sans": "^1.0",
         "webmozart/glob": "^4.7",
         "wikimedia/composer-merge-plugin": "2.1.0",
-        "yajra/laravel-pdo-via-oci8": "3.7.1"
+        "yajra/laravel-pdo-via-oci8": "3.7.1",
+        "monolog/monolog": "^3.9"
```

Comment:
Very minor point, but we keep our dependencies in alphabetical order; do you mind moving this up when time permits?